src/main/java/org/jvnet/hudson/update_center/Main.java

```diff
@@ -87,6 +93,15 @@
     @Option(name="-download",usage="Build mirrors.jenkins-ci.org layout")
     public File download = null;
 
+    @Option(name="-cache",usage="Cache target plugin")
+    public File cache = null;
+
+    @Option(name="-cacheAll",usage="Cache all version of plugin")
```

Comment:
```suggestion
    @Option(name="-cacheAll",usage="Cache all version of plugin instead of only the latest")
```

Reply:
I am not sure this has a legitimate use case, as the update center will not serve past releases. Why not e.g. `rsync` or `wget -r` from mirrors.jenkins.io instead to build your archive?

Reply:
Good idea. But one thing we might worth to do this is that keep one technology in the update-center solution might be better. For example, `rsync` or `wget` needs a command line which might dependent on OS platform.